src/data/yellowlist.txt

```diff
@@ -611,6 +613,7 @@ gatherer.wizards.com
 s.w.org
 ts.w.org
 ps.w.org
+wnyc.org
```

Comment:
Should this one be in the MDFP encompassing: wnycstudios.org, radiolab.org, wqxr.org, newsounds.org, thegreenespace.org, and thetakeaway.org? http://www.nypublicradio.org/

Reply:
Good question, I'll take a look. Is this good to go otherwise?

Reply:
Yes